product_docs/docs/livecompare/2/supported_technologies.mdx

```diff
@@ -34,11 +34,11 @@ in which context they can be used in LiveCompare.
 
 | Technology                     | Versions                        | Connections                 |
 | ------------------------------ | ------------------------------- | --------------------------- |
-| PostgreSQL                     | 10, 11, 12, 13 and 14 | Data and/or Output          |
-| EDB PostgreSQL Extended        | 10, 11, 12, 13 and 14      | Data and/or Output          |
-| EDB PostgreSQL Advanced (EPAS) | 11, 12, 13 and 14               | Data and/or Output          |
+| PostgreSQL                     | 10, 11, 12, 13, 14 and 15 | Data and/or Output          |
+| EDB PostgreSQL Extended        | 10, 11, 12, 13, 14 and 15     | Data and/or Output          |
+| EDB PostgreSQL Advanced (EPAS) | 11, 12, 13, 14 and 15               | Data and/or Output          |
 | pglogical                      | 2 and 3                         | Initial, Data and/or Output |
-| BDR                            | 1, 2, 3 and 4                   | Initial, Data and/or Output |
+| EDB Postgres Distributed       | 1, 2, 3, 4 and 5                   | Initial, Data and/or Output |
```

Comment:
again, I'm not sure we want to include 1 and 2 - we have no docs for those versions